Cocina/Cocina/Settings/Cells/EditProfileHeader.swift

```diff
@@ -9,9 +9,14 @@ import UIKit
 import SwiftUI
 import SnapKit
 
+protocol EditProfileHeaderDelegate {
+    func didTapEditImageButton()
+}
+
 class EditProfileHeader: UIView {
     
-    let size: CGFloat = 75
+    var delegate: EditProfileHeaderDelegate?
```

Comment:
Delegates should be weak to prevent strong reference cycles (aka leaking memory). Here's a short [article](https://medium.com/macoclock/delegate-retain-cycle-in-swift-4d9c813d0544) to read about this.